src/Sulu/Bundle/MediaBundle/Tests/Unit/Media/Manager/MediaManagerTest.php

```diff
@@ -520,11 +514,11 @@ public function testMediaPropertiesProvider(): void
         $this->assertSame(['key' => 'value'], $media->getProperties());
     }
 
-    public function provideGetByIds()
+    public static function provideGetByIds(): array
     {
-        $media1 = $this->createMedia(1);
-        $media2 = $this->createMedia(2);
-        $media3 = $this->createMedia(3);
+        $media1 = self::createMedia(1);
+        $media2 = self::createMedia(2);
+        $media3 = self::createMedia(3);
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes